location/http/server.go

```diff
@@ -0,0 +1,17 @@
+package http
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+)
+
+const serverPort = ":8888"
+
+func Run() {
+	http.Handle("/", GetRouter())
```

Comment:
port := ":8081"

	if os.Getenv("HTTP_PORT") != "" {
		port = os.Getenv("HTTP_PORT")
	}

Reply:
add code instead of const serverPort = ":8888"